docs/visualize.asciidoc

```diff
@@ -1,2 +1,131 @@
 [[visualize]]
-== Visualizing your Data
\ No newline at end of file
+== Visualize
+
+You can use Kibana's _Visualize_ app to design and create data visualizations. You can save these visualizations, use
+them individually, or combine visualizations into a _dashboard_. A visualization can be based on one of the following 
+data source types:
+
+* A new interactive search
+* A saved search
+* An existing saved visualization
+
+Visualizations are based on the {ref}/search-aggregations.html[aggregation] feature introduced in Elasticsearch 1. 
+
+[float]
+[[getting-started]]
+=== Creating a New Visualization
+
+To start the New Visualization wizard, click on the *Visualize* tab at the top left of the screen. Alternately, you can 
+click the *New Visualization* button image:images/K4NewDocument.png[New Document button] in the toolbar panel to the 
+right of the search bar. The wizard guides you through the following steps:
+
+[float]
+[[newvis01]]
+==== Step 1: Choose the Visualization Type
+
+The New Visualization wizard starts with the following screen:
+
+image:images/NewViz01.png[]
+
+The data table, Markdown, and metric elements aren't visualizations, but all the elements on the list can be included 
+in a <<dashboard,Dashboard>>.
+
+You can also open a saved visualization that you created earlier. The saved visualization selector includes a text 
+field to filter by visualization name and a link to the Object Editor, accessible through *Settings > Edit Saved 
+Objects*, to manage your saved visualizations.
+
+If your new visualization is a Markdown widget, selecting that type takes you to a text entry field where you enter the 
+text to display in the widget. For all other types of visualization, selecting the type takes you to data source 
+selection.
+
+[float]
+[[newvis02]]
+==== Step 2: Choose a Data Source
+
+You can choose a new or saved search to serve as the data source for your visualization. Searches are based on an 
+_index pattern_, which is a regular expression that matches the indices you want to visualize. After selecting _new 
```

Comment:
Section now reads "You can choose a new or saved search to serve as the data source for your visualization. Searches are associated with
an index or a set of indexes. When you select _new search_, select an index pattern from the drop-down to bring up the 
visualization editor. "

-- does that address the issue?